integrations/docker/images/chip-cirque-device-base/build.sh

```diff
@@ -40,10 +40,11 @@ VERSION=${DOCKER_BUILD_VERSION:-$(cat version)}
   Build and (optionally tag as latest, push) a docker image from Dockerfile in CWD
 
   Options:
-   --no-cache passed as a docker build argument
-   --latest   update latest to the current built version (\"$VERSION\")
-   --push     push image(s) to docker.io (requires docker login for \"$ORG\")
-   --help     get this message
+   --try-download  try to download latest image from dockerhub and skip whole build procedure if success
+   --no-cache      passed as a docker build argument
+   --latest        update latest to the current built version (\"$VERSION\")
```

Comment:
in the case of try-download, the operation seems reversed (we download, not build) so this '--latest` flag may not do update but rather it just downloads the latest version. Is this correct? I am unclear from the help text.

Reply:
The try-download will try to download the $VERSION from dockerhub, when --latest is also provided, it will tag the downloaded image latest.
This is useful for build containers for tests, since the Dockerfile are using `chip-cirque-device-base:latest` instead of some specific version.